Workbooks/Virtual Machines - Network Dependencies/Open Ports/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name": "Open Ports",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "microsoft.operationalinsights/workspaces", "order": 200 }
```

Comment:
This is the same order as the other template. Change it up or down based on priority.

Reply:
Updated order of workbooks
![image](https://user-images.githubusercontent.com/43890980/54391259-8940ad00-4661-11e9-8a2e-f4648796f578.png)